### ISSUE TITLE
[BUG]CXMM Radar Order book is empty when it's not

### ISSUE THREAD

**Issue by moraliang** [CoinAlpha/hummingbot#120]
**Describe the bug**
// Tried CXMM between Radar and binance, it showed the below error:
```
2019-04-25 11:17:53,016 - wings.clock - ERROR - Unexpected error running clock tick.
Traceback (most recent call last):
  File "wings/clock.pyx", line 95, in wings.clock.Clock.run_til
    child_iterator.c_tick(self._current_tick)
  File "hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx", line 422, in hummingbot.strategy.cross_exchange_market_making.cross_exchange_market_making.CrossExchangeMarketMakingStrategy.c_tick
    self.c_process_market_pair(market_pair, market_pair_to_active_orders[market_pair])
  File "hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx", line 496, in hummingbot.strategy.cross_exchange_market_making.cross_exchange_market_making.CrossExchangeMarketMakingStrategy.c_process_market_pair
    self.c_check_and_create_new_orders(market_pair, has_active_bid, has_active_ask)
  File "hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx", line 1107, in hummingbot.strategy.cross_exchange_market_making.cross_exchange_market_making.CrossExchangeMarketMakingStrategy.c_check_and_create_new_orders
    bid_profitable, ask_profitable = self.c_has_market_making_profit_potential(
  File "hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx", line 743, in hummingbot.strategy.cross_exchange_market_making.cross_exchange_market_making.CrossExchangeMarketMakingStrategy.c_has_market_making_profit_potential
    double maker_ask_price = maker_order_book.c_get_price_for_quote_volume(
  File "wings/order_book.pyx", line 322, in wings.order_book.OrderBook.c_get_price_for_quote_volume
    raise EnvironmentError(f"Requested quote volume {quote_volume} is beyond order book depth - no price quote is "
OSError: Requested quote volume 1000.0 is beyond order book depth - no price quote is possible
```

**Steps To Reproduce**
On Radar, use WETH-DAI. On binance, use ETHUSDC. Both sides have enough inventory. Then run the bot.

**Screenshots**
NA

**Release version**
0.4.0 with Ryan's Radar fix

**Attachments**
My configs as below: 
```
########################################################
###   Cross exchange market making strategy config   ###
########################################################

# The following configuations are only required for the
# cross exchange market making strategy

# Exchange and token parameters
maker_market: radar_relay
taker_market: binance
maker_market_symbol: WETH-DAI
taker_market_symbol: ETHUSDC

# Minimum profitability target required to place an order
# Expressed in decimals: 0.01 = 1% target profit
min_profitability: 0.0005

# Maximum order size in terms of quote currency
trade_size_override: 0.0

# Maximum aggregate amount of orders in quote currency
# that are allowed at a better price than Hummingbot's
# order before Hummingbot adjusts its order and pricing
top_depth_tolerance:

- -
    ^.+(USDT|USDC|USDS|DAI|PAX|TUSD)$
  - 1000
- - ^.+ETH$
  - 10
- - ^.+BTC$
  - 0.5
# Have Hummingbot actively adjust/cancel orders if necessary.
# If set to true, outstanding orders are adjusted if
# profitability falls below min_profitability.
# If set to false, outstanding orders are adjusted if
# profitability falls below cancel_order_threshold.
active_order_canceling: true

# If active_order_canceling = false, this is the profitability/
# loss threshold at which to cancel the order.
# Expressed in decimals: 0.01 = 1% target profit
cancel_order_threshold: -0.05

# An amount in seconds, which is the minimum duration for any
# placed limit orders. Default value = 130 seconds.
limit_order_min_expiration: 130.0
```

// Attach any relevant configuration files, logs, tx hashes, etc. here.

// NOTE: We understand If you do not want to publish your config/log files. You can instead send them privately to dev@coinalpha.com and drop a link of this issue in the email.
// WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!

**Comment by mifeng** [comment]
@moraliang when you have time, please run this again with version `0.6.0` - thanks!

**Comment by martinkou** [comment]
It could have happened that the WETH-DAI market was temporarily out of depth on the ask side. I think I'm gonna change the price / volume queries a bit to make it work / fail gracefully in case of not enough volume - rather than always raising an exception.

**Comment by martinkou** [comment]
Fixed with #290.
